frontend/src/pages/Profile/ProfileContainer.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-import useGetProfile from 'api/profile';
+import { useGetProfile } from 'api/profile';
 
 import Profile from './Profile';
 
 const ProfileContainer: React.FC = () => {
-  const { data, isSuccess, isLoading, isError } = useGetProfile();
+  const { data } = useGetProfile();
```

Comment:
Do we really need to remove those props?

Reply:
If we will safe it, ts will show warning 'yellow' message as like ` isSuccess, isLoading, isError dnt use`